packages/core/demo/data/bar.ts

```diff
@@ -93,6 +93,22 @@ export const simpleBarOptions = {
 	}
 };
 
+export const simpleBarCustomLegendOrderOptions = {
+	title: "Simple bar (custom legend order)",
+	axes: {
+		left: {
+			mapsTo: "value"
+		},
+		bottom: {
+			mapsTo: "group",
+			scaleType: "labels"
+		}
+	},
+	legend: {
+		order: ["Restocking", "Misc","Sold", "Qty", "More"]
```

Comment:
what if I pass in `order: ["Restocking", "Misc"]`, what legend items would I get then?

Reply:
i would expect it to order the legend "Restocking", "Misc" and then **all others** in the default order... 
but it does:  **all others** in the default order, then it does the custom ordered labels "Restocking", "Misc"

Reply:
I thought about this, but not sure if we need to take this into account.